docs/overview.md

```diff
@@ -0,0 +1,22 @@
+# ScalarDB Overview
+
+## What is ScalarDB?
+
+ScalarDB is an HTAP engine for diverse databases. It achieves ACID transactions and real-time analytics across databases to simplify the complexity of managing multiple databases. Existing solutions are designed to run only transactions or analytical queries across heterogeneous databases or both on homogeneous databases.
```

Comment:
Regarding the English expression, I will leave it to @jnmt 
I'd like to suggest constructing a story as a reader, shorten, common words, simple.
```suggestion
ScalarDB is a cross-database HTAP engine using a unique ACID transaction management.
There are several challenges in configuring an existing HTAP database.
- Transactions among different types of databases
- Trade-offs between performance and consistency
ScalarDB provides ACID transactions across multiple databases including non ACID compliant ones.
Analytical queries always answer consistent data without affecting performance.
This feature also relieves microservices developers from complicated designing tasks.
```
I just want to ask to re-arrange the composition, not the contents. Please replace my wrong contents into appropriate ones.

Reply:
> Regarding the English expression, I will leave it to @jnmt

@ochiai-kfj Probably, Josh?? : )

Reply:
@ochiai-kfj @josh-wong 
Thank you for the suggestions!
I got your ideas, but the suggested ones miss the analytics part (although it says HTAP), so it sounds a bit weird to me.
The suggestion is probably to use bullet points to clearly state the challenges, so let me update the paragraph based on that.